Dockerfile

```diff
@@ -1,13 +1,13 @@
 FROM golang:1.11.5-alpine as builder
 ENV GO111MODULE=on
-COPY . /go/src/github.com/linode/docker-volume-linode
-WORKDIR /go/src/github.com/linode/docker-volume-linode
-RUN set -ex \
-    && apk update && apk add git \
-    && apk add --no-cache --virtual .build-deps \
-    gcc libc-dev \
-    && go install --ldflags '-extldflags "-static"' \
-    && apk del .build-deps
+COPY . /src
+WORKDIR /src
+RUN apk update
```

Comment:
The chaining of commands on a single `RUN` (as this was) reduce the number of layers (size) of the docker image.  Also, because of image caching, the `apk update` image cache could be weeks old and refer to a version of `git` which is no longer in the repository.  I think, in this case, it is better to leave these commands in a single `RUN`.

Reply:
This is a multi-stage build. The only layer in the build is the last COPY after the second FROM. The advantage of having separate RUN lines is that they cache separate.

Reply:
This is a multi-stage build. The only layer in the build is the last COPY after the second FROM. The advantage of having separate RUN lines is that they cache separate.

Reply:
I was referring to this: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get
> Always combine RUN apt-get update with apt-get install in the same RUN statement.
> Using apt-get update alone in a RUN statement causes caching issues and subsequent apt-get install instructions fail.

I assume this would apply with `apk` equally.

Reply:
That is interesting.  Thanks for the tip.  Will adjust!